setup.py

```diff
@@ -7,7 +7,7 @@
 
 inst_reqs = [
     "rasterio",
-    "pystac>=1.0.0,<1.1.0",
+    "pystac>=1.0.0",
```

Comment:
let's restrict from `1.x` to `2.x` https://github.com/stac-utils/pystac#versions